modules/local/csvtk_split.nf

```diff
@@ -1,7 +1,7 @@
 process CSVTK_SPLIT {
     label 'process_low'
 
-    conda (params.enable_conda ? "bioconda::csvtk=0.23.0" : null)
+    conda (params.enable_conda ? "conda-forge::sed=4.7 bioconda::csvtk=0.23.0" : null)
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
         'https://depot.galaxyproject.org/singularity/csvtk:0.23.0--h9ee0642_0' :
```

Comment:
Shouldn't this be replayed with a mulled container?

Reply:
I guess you mean because of `sed`? It works using `docker` as it is but when using `-profile conda` locally I got an error because of `sed` which was solved by adding the package.